### ISSUE TITLE
Notification icon needs a label in site-wide navigation

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/knowledge-commons-profiles#376]
Area of site: Commons
URL: https://hcommons-staging.org/

**Description**
The notification icon that appears in the site header does not have a label. 

**Recommendation**
Either of these should fix the issue:

1. Add a `title` attribute to this
2. Add an `aria-label` attribute to this
